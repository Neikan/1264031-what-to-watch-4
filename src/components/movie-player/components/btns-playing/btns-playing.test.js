// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";

// Импорт компонентов
import BtnsPlaying from "./btns-playing.jsx";


describe(`Test BtnsPlaying component`, () => {
  test(`BtnsPlaying component is created and rendered correctly when isPlaying=true`, () => {
    const tree = renderer.create(
        <BtnsPlaying
          isPlaying={true}
          onPlayingChange={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });


  test(`BtnsPlaying component is created and rendered correctly  when isPlaying=false`, () => {
    const tree = renderer.create(
        <BtnsPlaying
          isPlaying={false}
          onPlayingChange={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
