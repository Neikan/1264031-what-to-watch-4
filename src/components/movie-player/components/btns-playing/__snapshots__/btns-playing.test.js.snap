// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test BtnsPlaying component BtnsPlaying component is created and rendered correctly  when isPlaying=false 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;

exports[`Test BtnsPlaying component BtnsPlaying component is created and rendered correctly when isPlaying=true 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlinkHref="#pause"
    />
  </svg>
  <span>
    Pause
  </span>
</button>
`;
